swabbie-aws/src/main/kotlin/com/netflix/spinnaker/swabbie/aws/snapshots/AmazonSnapshotHandler.kt

```diff
@@ -172,35 +171,9 @@ class AmazonSnapshotHandler(
   private fun checkIfImagesExist(candidates: List<AmazonSnapshot>, params: Parameters) {
     log.info("Checking for existing images for {} snapshots. Params: {} ", candidates.size, params)
     candidates.forEach { snapshot ->
-      if (snapshot.description == null) {
-        snapshot.set(NAIVE_EXCLUSION, true)
-        return
-      }
-
-      if (descriptionIsFromAutoBake(snapshot.description)) {
-          // if description contains these keys then it's most likely an auto-captured snapshot.
-        if (usedResourceRepository.isUsed(SNAPSHOT, snapshot.snapshotId, "aws:${params.region}:${params.account}")) {
-          snapshot.set(IMAGE_EXISTS, true)
-        }
-      } else {
-        snapshot.set(NAIVE_EXCLUSION, true)
+      if (usedResourceRepository.isUsed(SNAPSHOT, snapshot.snapshotId, "aws:${params.region}:${params.account}")) {
+        snapshot.set(IMAGE_EXISTS, true)
       }
     }
   }
-
-  // todo eb: generalize this netflix-specific pattern
-  fun descriptionIsFromAutoBake(description: String): Boolean {
```

Comment:
Where is this check happening instead , do we not need this for `nflx` case ?

Reply:
never mind , did the read the description before.

Reply:
This check is now replaced by an Attribute rule matching on the description. Having it hard coded here doesn’t allow non Netflix users to use this handler.